# HG changeset patch
# User Rik <rik@octave.org>
# Date 1447047284 28800
#      Sun Nov 08 21:34:44 2015 -0800
# Node ID dc2be248596881d55b71aec23a2a3373c345c1ec
# Parent  655816377845d7e616923e76e5033f375f9898bb
NEWS: Update with changes to quadl function.

* NEWS: Add note about changes to quadl function.  Re-wrap lines to 80
characters.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -5,25 +5,24 @@ Summary of important user-visible change
     in numbers.  This facilitates writing more legible code by using '_' as
     a thousands separator or to group nibbles into bytes in hex constants.
 
     Examples: 1_000_000 == 1e6 or 0xDE_AD_BE_EF
 
  ** The parser has been extended to understand binary numbers which begin
     with the prefix '0b' or '0B'.  The value returned is Octave's default
     numeric class of double, not at unsigned integer class.  Therefore numbers
-    greater than flintmax (2^53) will lose some precision.
+    greater than flintmax, i.e., 2^53, will lose some precision.
 
     Examples: 0b101 == 5 or 0B1100_0001 == 0xC1
 
- ** The default colormap is now set to 'viridis' which is also
-    the default colormap in matplotlib. This new colormap fixes
-    some of the main issues with the old default colormap 'jet'
-    such as its bad 'luminance profile' and is also more similar
-    to Matlab's new default colormap 'parula'.
+ ** The default colormap is now set to "viridis" which is also the default
+    colormap in matplotlib.  This new colormap fixes some of the main issues
+    with the old default colormap "jet" such as its bad "luminance profile"
+    and is also more similar to Matlab's new default colormap "parula".
 
  ** Octal ('\NNN') and hex ('\xNN') escape sequences in single quoted
     strings are now interpreted by the function do_string_escapes().
     The *printf family of functions now supports octal and hex escape
     sequences in single-quoted strings for Matlab compatibility.
 
  ** Special octal and hex escape sequences for the pattern and replacement
     strings in regular expressions are now interpreted for Matlab compatibility.
@@ -48,16 +47,22 @@ Summary of important user-visible change
 
  ** The griddata function no longer plots the interpolated mesh if no output
     argument is requested, instead the vector or array of interpolated values
     is always returned for Matlab compatibility.
 
  ** The surfnorm function now returns unnormalized (magnitude != 1) normal
     vectors for compatibility with Matlab.
 
+ ** The quadl function now uses an absolute, rather than relative, tolerance
+    for Matlab compatibility.  The default tolerance is 1e-6 which may result
+    in lower precision results than previous versions of Octave which used
+    eps as the relative tolerance.  The quadl function has also been extended
+    to return a second output with the total number of function evaluations.
+
  ** Other new functions added in 4.2:
 
       psi
       odeset
       odeget
       ode45
 
  ** Deprecated functions.
